src/dbSta/include/db_sta/dbNetwork.hh

```diff
@@ -383,6 +394,12 @@ class dbNetwork : public ConcreteNetwork
   bool hasMembers(const Port* port) const override;
   Port* findMember(const Port* port, int index) const override;
   PortMemberIterator* memberIterator(const Port* port) const override;
+  void removeDriverFromCache(const Net* net, const Pin* drvr);
+
+  ////////////////////////////////////////////////////////////////
+  // Debug functions
+  PinInfo getPinInfo(const Pin* pin) const;
+  void dumpNetDrvrPinMap() const;
```

Comment:
Can these be private?  If so, PinInfo  can be private too.  It doesn't feel like a public API.